README.md

```diff
@@ -93,3 +93,7 @@ We keep a list of [technical FAQs in the Kedro documentation](https://docs.kedro
 ## How can I cite Kedro?
 
 If you're an academic, Kedro can also help you, for example, as a tool to solve the problem of reproducible research. Use the "Cite this repository" button on [our repository](https://github.com/kedro-org/kedro) to generate a citation from the [CITATION.cff file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files).
+
+## Python version support policy
+* The core [Kedro Framework](https://github.com/kedro-org/kedro) supports all Python versions that are actively maintained by [the Python Software Foundation](https://www.python.org/psf-landing/). When a [Python version reaches end of life](https://devguide.python.org/versions/#versions), support for that version is dropped from Kedro. This is not considered a breaking change.
```

Comment:
Splitting hairs here but I'm not 100 % sure that the PSF maintains CPython, maybe we should be saying here "by the CPython core team". May I summon @pablogsal ? 🖤

Reply:
The most correct statement is that "The CPython team maintains CPython" as the PSF mainly holds the trademarks rights.

Reply:
Thanks a bunch 🙏🏽

Reply:
Probably splitting hairs, but we _attempt_ to support the newer versions, but sometimes it takes a while. Does it make sense to distinguish that we're talking about policy for older versions?

Reply:
I think going forward and especially since splitting datasets it will be much easier to support newer versions right? it's just that we haven't gotten around adding 3.12 to our CI yet, I believe